root/static/scripts/artist.js

```diff
@@ -1,12 +1,14 @@
 /*
- * @flow strict-local
+ * @flow
```

Comment:
Doesn't seem needed, unless I'm missing something? It's strict-local, not strict.

Reply:
Right, I copied this from another branch where it was needed. Will revert it from here.